test/DataDryadAPI.jl

```diff
@@ -4,6 +4,16 @@ using Base.Test
 using ReferenceTests
 
 @testset "DataDryadAPI test" begin
+    # @testset "DataDryadWeb Ecology" begin
```

Comment:
Why is this one commented out?

Reply:
For some weird reason, I am able to manually download the files specified in the urls, but the test are not being able to do so. I'll have to try again.

Reply:
Is that everywhere not able to or just A.V being weird.

Reply:
Locally too. I don't remember the exact error, I'll try and revert back to you.

Reply:
```
Integration Test: Test Failed
  Expression: length(collect(readdir(@datadep_str("Data from Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation")))) > 0
   Evaluated: 0 > 0
```